src/Key.re

```diff
@@ -0,0 +1,56 @@
+type t =
+  | Character(char)
+  | Function(int)
+  | NumpadDigit(int)
+  | Escape
+  | Down
+  | Up
+  | Left
+  | Right
+  | Tab
+  | PageUp
+  | PageDown
+  | Return
+  | Space
+  | Delete
+  | Pause
+  | Home
+  | End
+  | Backspace
+  | CapsLock
+  | Insert
+  | NumpadMultiply
+  | NumpadAdd
+  | NumpadSeparator
+  | NumpadSubtract
+  | NumpadDecimal
+  | NumpadDivide;
+
+let to_string =
```

Comment:
Why is this suddenly snake cased?

Reply:
Oops, was spending a lot of time looking at OCaml parsing code